File: src/main/java_ex/vladsaif/syncedit/plugin/lang/transcript/psi/TranscriptElementType.java
package vladsaif.syncedit.plugin.lang.transcript.psi;

import com.intellij.psi.tree.IElementType;
import org.jetbrains.annotations.NotNull;
import vladsaif.syncedit.plugin.lang.transcript.TranscriptLanguage;

public class TranscriptElementType extends IElementType {
    public TranscriptElementType(@NotNull String debugName) {
        super(debugName, TranscriptLanguage.getInstance());
    }
}

File: src/main/kotlin/com/github/recognized/screencast/editor/lang/transcript/lexer/TranscriptElementType.kt
package com.github.recognized.screencast.editor.lang.transcript.lexer

import com.github.recognized.screencast.editor.lang.transcript.psi.TranscriptLanguage
import com.intellij.psi.tree.IElementType

class TranscriptElementType(debugName: String) : IElementType(debugName, TranscriptLanguage)